static-analysis/teammates-eslint.yml

```diff
@@ -7,17 +7,7 @@ env:
 extends:
     - airbnb-base
 
-parserOptions:
-    ecmaVersion: 2015
```

Comment:
I think @LiHaoTan suggested this since the base version (airbnb) already has support for ES6+ ?
https://github.com/airbnb/javascript/tree/master/packages/eslint-config-airbnb-base